intro-Linux/title.tex

```diff
@@ -1,44 +1,36 @@
-\pagestyle{empty}
-
 \begin{center}
 
-\includegraphics*[width=8truecm]{logo_vsc}
+\includegraphics*[width=12truecm]{logo_vsc_2}
 
-\vspace*{1.5\baselineskip}
+\vspace*{6\baselineskip}
 
 \Huge \strong{Linux Tutorial} \\
-\LARGE Last updated: DATEPLACEHOLDER
+\LARGE Last updated: September 20 2018
```

Comment:
Same here, please keep `DATEPLACEHOLDER` (and maybe include a comment to make it clear it should stay)?

Reply:
Done.